src/frontend/src/handler/explain.rs

```diff
@@ -29,9 +30,12 @@ use crate::session::OptimizerContext;
 pub(super) fn handle_explain(
     context: OptimizerContext,
     stmt: Statement,
-    _verbose: bool,
+    verbose: bool,
 ) -> Result<PgResponse> {
     let session = context.session_ctx.clone();
+    session
+        .set_config(EXPLAIN_VERBOSE_STRING, &verbose.to_string())
+        .unwrap();
```

Comment:
Session config is kept for whole session, so I think it's better to put it execution context.

Reply:
Agree, I will move it to OptimizerContext